src/safeposix/syscalls/sys_calls.rs

```diff
@@ -216,6 +216,11 @@ impl Cage {
 
         //may not be removable in case of lindrustfinalize, we don't unwrap the remove result
         interface::cagetable_remove(self.cageid);
+        
+        // Trigger SIGCHLD
+        if self.cageid != self.parent {
+            interface::kill_inner(self.parent, 17);
```

Comment:
I think we should add the signal numbers as constants in sys_constants, will make this more readable and potentially be helpful for other things.